charts/cla/values.yaml

```diff
@@ -0,0 +1,145 @@
+# Default values for cla-assistant.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+claAssistant:
+  host:
+  port:
+  hostPort:
+  protocol:
+
+github:
+  secret: "a3da5d4a65ds"
+  password: "a3da5d4a65ds"
+  token: "a3da5d4a65ds"
+
+mongodb:
+  mongodbRootPassword:
+  service:
+    port: 56985
+  mongodbUsername: mongol
+  mongodbPassword: "password"
+  mongodbDatabase: cla
+  replicaSet:
+    enabled: true
+    replicas:
+      secondary: 1
+      arbiter: 1
+    pdb:
+      enabled: true
+      minAvailable:
+        secondary: 1
+        arbiter: 1
+      maxUnavailable:
+        secondary: 1
+        arbiter: 1
+  resourcesArbiter:
+    limits:
+      cpu: 500m
+      memory: 512Mi
+    requests:
+      cpu: 100m
+      memory: 256Mi
+  securityContext:
+    enabled: true
+    fsGroup: 1001
+    runAsUser: 1001
+  livenessProbe:
+    enabled: true
+    initialDelaySeconds: 30
+    periodSeconds: 10
+    timeoutSeconds: 5
+    failureThreshold: 6
+    successThreshold: 1
+  readinessProbe:
+    enabled: true
+    initialDelaySeconds: 5
+    periodSeconds: 10
+    timeoutSeconds: 5
+    failureThreshold: 6
+    successThreshold: 1
+
+  metrics:
+    enabled: true
+    image:
+      registry: docker.io
+      repository: bitnami/mongodb-exporter
+      tag: 0.10.0-debian-9-r77
+      pullPolicy: IfNotPresent
+    livenessProbe:
+      enabled: true
+      initialDelaySeconds: 15
+      periodSeconds: 5
+      timeoutSeconds: 5
+      failureThreshold: 3
+      successThreshold: 1
+    readinessProbe:
+      enabled: true
+      initialDelaySeconds: 5
+      periodSeconds: 5
+      timeoutSeconds: 1
+      failureThreshold: 3
+      successThreshold: 1
+    podAnnotations:
+      prometheus.io/scrape: "true"
+      prometheus.io/port: "9216"
+    serviceMonitor:
+      enabled: true
+
+
+replicaCount: 2
+
+image:
+  repository: web3f
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: []
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name:
+
+podSecurityContext: {}
```

Comment:
There are a lot of values that are not used for this simple chart. IMO we should remove them from here and from the deployment, and add them in case we are going to use them, WDYT?

Reply:
I will do the cleanup as a final step. still doing tests with values.